### PR TITLE
Prepare 1.7.0

### PR DESCRIPTION
**Pull request by daniel-mohemian**
I additionally added all contributors to changelog entries.

I'll definitely extend the changelog to include _all_ contributors! See #144.

Let me know if it's weird to include ourselves. 😅

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,29 +17,33 @@ Start each entry with one of the following keywords:
...
 
 ## [1.5.0] - 2019-01-23
 
-- Changes the way elliptic curve keys are decoded to work around [#86](https://github.com/airsidemobile/JOSESwift/issues/86) until [#120](https://github.com/airsidemobile/JOSESwift/pull/120) is merged ([#137](https://github.com/airsidemobile/JOSESwift/pull/137))
-- Adds a changelog ([#136](https://github.com/airsidemobile/JOSESwift/pull/136))
-- Changes travis to use builtin homebrew addon ([#133](https://github.com/airsidemobile/JOSESwift/pull/133))
-- Security: Change fastlane version to fix [#129](https://github.com/airsidemobile/JOSESwift/issues/129) ([#130](https://github.com/airsidemobile/JOSESwift/pull/130))
-- Adds support for elliptic curve algorithms for JWS and elliptic curve keys for JWK ([#88](https://github.com/airsidemobile/JOSESwift/pull/88))
+- Changes the way elliptic curve keys are decoded to work around [#86](https://github.com/airsidemobile/JOSESwift/issues/86) until [#120](https://github.com/airsidemobile/JOSESwift/pull/120) is merged ([#137](https://github.com/airsidemobile/JOSESwift/pull/137)) via  via [@daniel-mohemian](https://github.com/daniel-mohemian)
```
Comment (nathan-mohemian) [inline CHANGELOG.md]:
```suggestion
- Changes the way elliptic curve keys are decoded to work around [#86](https://github.com/airsidemobile/JOSESwift/issues/86) until [#120](https://github.com/airsidemobile/JOSESwift/pull/120) is merged ([#137](https://github.com/airsidemobile/JOSESwift/pull/137)) via [@daniel-mohemian](https://github.com/daniel-mohemian)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,29 +17,33 @@ Start each entry with one of the following keywords:
...
+- Adds a changelog ([#136](https://github.com/airsidemobile/JOSESwift/pull/136))  via [@daniel-mohemian](https://github.com/daniel-mohemian)
```
Comment (nathan-mohemian) [inline CHANGELOG.md]:
```suggestion
- Adds a changelog ([#136](https://github.com/airsidemobile/JOSESwift/pull/136)) via [@daniel-mohemian](https://github.com/daniel-mohemian)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,29 +17,33 @@ Start each entry with one of the following keywords:
...
+- Changes travis to use builtin homebrew addon ([#133](https://github.com/airsidemobile/JOSESwift/pull/133))  via [@stephengroat](https://github.com/stephengroat)
```
Comment (nathan-mohemian) [inline CHANGELOG.md]:
```suggestion
- Changes travis to use builtin homebrew addon ([#133](https://github.com/airsidemobile/JOSESwift/pull/133)) via [@stephengroat](https://github.com/stephengroat)
```

**Comment by daniel-mohemian** [comment]
Thanks @nathan-mohemian!

**Pull request merged**